lib/utils/addresses.dart

```diff
@@ -1,12 +1,14 @@
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 
-String defaultCommunity = DotEnv().env['DEFAULT_COMMUNITY_CONTRACT_ADDRESS'].toLowerCase();
+String defaultCommunityAddress = DotEnv().env['DEFAULT_COMMUNITY_CONTRACT_ADDRESS'].toLowerCase();
+
+final String daiTokenAddress = '0x6b175474e89094c44da98b954eedeac495271d0f';
```

Comment:
it should be a different address on ropsten. move this to config?

Reply:
yeah you right.